iron-ajax.html

```diff
@@ -317,9 +317,11 @@
      * @return {Object}
      */
     get requestHeaders() {
-      var headers = {
-        'Content-Type': this.contentType
-      };
+      var headers = {};
+
+      if ( this.contentType !== 'false' ) {
```

Comment:
A user can set it like the jquery $ajax() function.  By not specifying a
content type in the xhr request one is generated in the form of
"multipart/form-data boundary=<someGeneratedDelimiter>". This is required
for proper payload parsing of FormData js objects. The main advantage of
this, for me at least, is PHP will now correctly populate its $_FILES super
global variable when a file exists in the payload.
On Jul 8, 2015 12:34 AM, "Peter Burns" <notifications@github.com> wrote:

> In iron-ajax.html
> <https://github.com/PolymerElements/iron-ajax/pull/73#discussion_r34099339>
> :
>
> > @@ -317,9 +317,11 @@
> >       * @return {Object}
> >       */
> >      get requestHeaders() {
> > -      var headers = {
> > -        'Content-Type': this.contentType
> > -      };
> > +      var headers = {};
> > +
> > +      if ( this.contentType !== 'false' ) {
>
> How would this.contentType end up with the string value 'false'?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/PolymerElements/iron-ajax/pull/73/files#r34099339>.
>